### ISSUE TITLE
[Feature] - Dark mode 

### ISSUE THREAD

**Issue by kabir0x23** [cryptoverseWeb3/Cryptoverse#13]
- Dark mode:
  - Below screenshots are in the darkmode because of an extension Dark-reader.
  - Floating dark mode toggle can be a good feature.
- Screenshots:
![image](https://user-images.githubusercontent.com/44284877/178080228-3b6618ef-cee4-4026-9fad-75d5bfda193b.png)
![image](https://user-images.githubusercontent.com/44284877/178080290-da4563cb-f022-4d8a-9fea-e57edaa41c79.png)

**Comment by ArunBohra12** [comment]
Hi @kabir0x23, I would like to work on this enhancement. Can you please assign it to me?

**Comment by kabir0x23** [comment]
> Hi @kabir0x23, I would like to work on this enhancement. Can you please assign it to me?

Hey @ArunBohra12,
- we are using [antd](https://ant.design) for designing, like boxes and text.
- and I am assigning you this _issue._
- and first, can you first contribute to this project: https://github.com/CryptoverseWeb3/git-github-practice

**Comment by ArunBohra12** [comment]
I have read the project [https://github.com/CryptoverseWeb3/git-github-practice](https://github.com/CryptoverseWeb3/git-github-practice)

And before getting started, I want to ask if we need the color theme the same as the image provided above in the thread or can I propose one?

**Comment by kabir0x23** [comment]
> I have read the project https://github.com/CryptoverseWeb3/git-github-practice
> 
> And before getting started, I want to ask if we need the color theme the same as the image provided above in the thread or can I propose one?

You can make the proposed one.

**Comment by JadugarKalakaboola** [comment]
Hi kabir, I would like to work on this issue too :)

**Comment by kabir0x23** [comment]
Hello @ArunBohra12 ?
any progress, are you working on this issue?

**Comment by ArunBohra12** [comment]
Yes I'm working on it, I've been a little busy so haven't accomplished much, if @JadugarKalakaboola  wants to work on it then he can.

**Comment by kabir0x23** [comment]
> Yes I'm working on it, I've been a little busy so haven't accomplished much, if @JadugarKalakaboola wants to work on it then he can.
Okay, you can take your time, I will assign him another issue

**Comment by kabir0x23** [comment]
@JadugarKalakaboola 
You can work on this issue 
https://github.com/CryptoverseWeb3/CryptoverseWeb3.com/issues/8

it's the same

**Comment by ArunBohra12** [comment]
Ok, thanks!
